code/game/machinery/ARES/ARES_records.dm

```diff
@@ -49,6 +49,15 @@
 	src.details = details
 	src.user = user
 
+/datum/ares_record/flight
+	record_name = ARES_RECORD_FLIGHT
+
+/datum/ares_record/flight/New(details, user)
+	time = worldtime2text()
+	src.title = "Flight Log"
```

Comment:
```suggestion
	title = "Flight Log"
```
you only need src to disambiguate between local vars and object vars, which this is not